pkg/controller/windowsmachine/nodeconfig/nodeconfig.go

```diff
@@ -136,6 +136,10 @@ func (nc *nodeConfig) Configure() error {
 	if err := nc.configureNetwork(); err != nil {
 		return errors.Wrap(err, "configuring node network failed")
 	}
+	// Start Windows metrics exporter service
+	if err := nc.Windows.ConfigureWindowsExporter(); err != nil {
+		return errors.Wrapf(err, "error configuring windows_exporter for %s", nc.node.GetName())
```

Comment:
Please use `nc.ID()` not `GetName()`, in order to be consistant with the other log messages